packages/web-component/src/lib/types.ts

```diff
@@ -125,3 +125,9 @@ export interface Context {
   loginId?: string;
   code?: string;
 }
+
+export interface ILogger {
+  info(title: string, description: string, state: any): void;
+  warn(title: string, description?: string): void;
+  error(title: string, description?: string, params?: any): void;
```

Comment:
I don't think this is the proper interface, it should be `info(message?: any, ...optionalParams: any[])` , like the console interface (also for warn/error)

maybe even use the `Parameters<console.info>` or  have the entire `ILogger` pick the info,warn and error from `console`

Reply:
i do want to differentiate between our logger (that has state with possible types we can add for example) rather then console.info(params) we are actually never used by us. The console still implements the same method signatures

Reply:
i did change the error signature to better match

Reply:
I don't think that this differentiation helps, it will just add redundant constrain to the developer which is not needed
I would also do this for the other errors if possible